src/collectd.conf.pod

```diff
@@ -2855,6 +2855,12 @@ Memory subsystem rail (uVolts).
 
 =back
 
+=item B<WsrepStats> I<true|false>
```

Comment:
I assume placing this between the mic and memory plugin sections is a mistake. Can you correct this please @XANi ?

Also, there's no obvious way to know if the option is enabled or disabled by default. Can you amend the doc to mention this detail ?

I realise none of the other boolean options to this plugin document this either. I'll add this to the bugfix branches.